packages/client/src/services/persistantStorage.ts

```diff
@@ -0,0 +1,31 @@
+const NAMESPACE = "__REACT_TDD_POC";
+const STRINGIFIED_OBJECT = "{}";
+
+interface StorageData {
```

Comment:
don't you think it's a bit YAGNI? all we need is a cart id

Reply:
since localStorage is the mother of all global-persistent scopes, we might break other peoples' apps using such a common name.
Also, I don't really like the fact that your business logic knows of the implementation of its dependent data persistency,.

Also, I have a typo in the services' name 🙈

Reply:
we're not a library, we're a demo app